test_data/series7_device_config.yaml

```diff
@@ -94,3 +94,54 @@ disabled_cell_bel_map:
      - TFF
      - IFF
      - OUTFF
+disabled_site_pips:
+  - bels:
+     - A6LUT
+     - B6LUT
+     - C6LUT
+     - D6LUT
+    ipin: A6
+    opin: O6
+bel_chains:
+- name: CARRY_CHAIN
+  source:
+    cell_port: CO[3]
+    bel_port: CO3
+  sink:
+    cell_port: CI
+    bel_port: CIN
+  sites:
+    - SLICEL
+    - SLICEM
+  cells:
+    - CARRY4
+  chain_drivers: # Place driver cell of S port to a proper BEL
+  - ports:
+    - name: S[0]
+      bels:
+        - AFF
+        - A6LUT
+    - name: S[1]
+      bels:
+        - BFF
+        - B6LUT
+    - name: S[2]
+      bels:
+        - CFF
+        - C6LUT
+    - name: S[3]
+      bels:
+        - DFF
+        - D6LUT
+    in_out_cells:
```

Comment:
I'm wondering if a cell type and pin combination is more useful than just a cell type

Reply:
For context, the idea behind the `in_out_cells` was mainly to bind a cell to a cluster in case that cell is connected to both in and out ports of the main cluster cell (in this case the carry).
I'll need to think of a better representation for this and yeah I think a pin combination is actually the way to go